test/cpp/end2end/xds/xds_override_host_end2end_test.cc

```diff
@@ -94,7 +94,6 @@ class OverrideHostTest : public XdsEnd2endTest {
               std::string(it->second).c_str());
       values.emplace_back(ParseCookie(it->second));
       EXPECT_FALSE(values.back().value.empty());
-      EXPECT_THAT(values.back().attributes, ::testing::Contains("HttpOnly"));
```

Comment:
I think it's better to leave this here, so that we're checking this in every test.  We want this attribute to always be set, no matter what else we may be setting.